tests/system/test_unreachable.py

```diff
@@ -0,0 +1,91 @@
+""" Test using root marathon.
+    This test suite is used to test unreachable strategy configurations on live clusters.
+"""
+
+import apps
+import common
+import json
+import os
+import pytest
+import retrying
+import shakedown
+import time
+import uuid
+
+from dcos.mesos import DCOSClient
+from dcos import marathon, errors, mesos
+from datetime import timedelta
+
+
+from shakedown import dcos_version_less_than, marthon_version_less_than
+from fixtures import wait_for_marathon_and_cleanup
+
+
+@pytest.fixture(scope="function")
+def marathon_service_name():
+    return "marathon"
+
+
+def setup_module(module):
+    common.cluster_info()
+
+
+def teardown_module(module):
+    common.clean_up_marathon()
+
+# This can not be run currently as `shakedown -q tests/system/test_unreachable.py`
+# They currently need a 20 min delay prior to running each one separately or you get
+# inappropriate results.
+# TO RUN:  shakedown -q tests/system/test_unreachable.py::test_unreachable_within_inactive_time[360-360]
+@pytest.mark.parametrize("inactive_sec, expunge_sec", [
```

Comment:
What is the reason for the different parameters? What are we trying to verify?

Reply:
unreachable timings from the customers perspective do not work the way I would expect them to....   This is gather a number of different combinations to evaluate the details.

Reply:
Got it.